libkbfs/folder_branch_ops.go

```diff
@@ -1723,13 +1723,16 @@ func (fbo *folderBranchOps) syncBlockAndCheckEmbedLocked(ctx context.Context,
 		return path{}, DirEntry{}, nil, err
 	}
 
-	// do the block changes need their own blocks?
-	bsplit := fbo.config.BlockSplitter()
-	if !bsplit.ShouldEmbedBlockChanges(&md.data.Changes) {
-		err = fbo.unembedBlockChanges(ctx, bps, md, &md.data.Changes,
-			uid)
-		if err != nil {
-			return path{}, DirEntry{}, nil, err
+	// Do the block changes need their own blocks?  Unembed only if
```

Comment:
I feel like there are a bunch of places we're doing this check, and we could easily miss another one. Maybe we should add a check in MDOps that any MD that is about to be sent to the server should have unembedded changes or !ShouldEmbedBlockChanges should hold. What do you think?